fast_llm/models/gpt/conversion.py

```diff
@@ -699,8 +699,10 @@ class DiffusionDreamHuggingfaceCheckpointHandler(CustomModelingExportMixin, Comm
     def _create_config_converters(cls) -> list[ParamConverter]:
         return super()._create_config_converters() + [
             # From Qwen2HuggingfaceCheckpointHandler - Change architectures to DiffusionDream
+            ConstantExportParamConverter(export_names=(("architectures",),), export_value=["DreamModel"]),
```

Comment:
These are almost the same as existing converters, we should inherit instead of making near-identical copies